app/build.gradle.kts

```diff
@@ -135,6 +135,7 @@ android {
 
 dependencies {
     implementation(project(mapOf("path" to ":data")))
+    implementation(project(mapOf("path" to ":compose")))
     // TODO: Find a way to make sure we are aware of out-of-date versions of any static aars/jars in /libs. Manually check for any updates at/prior to dev signoff.
     implementation(fileTree(mapOf("dir" to "libs", "include" to listOf("*.jar"))))
```

Comment:
Sometimes, I will skip this and specifically include include a reference to each lib in libs dir.  That avoids the danger of someone "slipping" a dangerous lib into the directory and having it compile into app.

Reply:
@BottleRocket-Colin That is a good point! I'll make a note of it to be adjusted/removed in a future PR along with guidance in comments to add an implementation for each lib :)